codebuild/bin/s2n_open_fds_test.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
Let's add the copyright/license header.

Let's also add some top level/module documentation about what this file does and why it exists.